File: java/amp-commanding/src/main/java/amp/commanding/DefaultCommandSender.java
package amp.commanding;


import java.util.List;

import amp.messaging.IMessageProcessor;
import amp.messaging.MessageSender;
import cmf.bus.IEnvelopeSender;


/**
 * Created with IntelliJ IDEA.
 * User: jar349
 * Date: 5/1/13
 */
public class DefaultCommandSender extends MessageSender implements ICommandSender {

    public DefaultCommandSender(IEnvelopeSender envelopeSender) {
        super(envelopeSender);
    }

    public DefaultCommandSender(IEnvelopeSender envelopeSender, List<IMessageProcessor> processorChain) {
        super(envelopeSender, processorChain);
    }
}

File: dotnet/source/amp.commanding/DefaultCommandSender.cs
using System.Collections.Generic;
using amp.messaging;
using cmf.bus;

namespace amp.commanding
{
    public class DefaultCommandSender : MessageSender, ICommandSender
    {
        public DefaultCommandSender(IEnvelopeSender envelopeSender) 
            : base(envelopeSender)
        {
        }

        public DefaultCommandSender(IEnvelopeSender envelopeSender, List<IMessageProcessor> processingChain) 
            : base(envelopeSender, processingChain)
        {
        }
    }
}
